backend/app/api/v1/endpoints/cpt/cptJobs.py

```diff
@@ -102,7 +102,7 @@ async def jobs(
         "endDate": end_date.__str__(),
         "results": results_df.to_dict("records"),
         "total": jobsCount,
-        "offset": offset + size,
+        "offset": offset + size if size else 0,
```

Comment:
In some APIs (and you added some more with this PR), you're protecting against having `offset` set to a value without `size`. That's not the case here, and this expression isn't considering the value of `offset`. So `if size` we add `offset + size` (regardless of `offset`) or else use `0`.

Is that what you meant? Should you be raising the `HTTPException` here `if offset and not size`?

```python
>>> offset = 10
>>> size = None
>>> offset + size if size else 0
0
>>> offset + (size if size else 0)
10
```